cpp/unreal_plugins/SimulationController/Source/SimulationController/ImuSensor.cpp

```diff
@@ -18,10 +16,13 @@
 
 #include "CoreUtils/Assert.h"
 #include "CoreUtils/Config.h"
+#include "CoreUtils/Log.h"
 #include "SimulationController/TickEventComponent.h"
 
 ImuSensor::ImuSensor(UPrimitiveComponent* primitive_component)
 {
+    SP_LOG_CURRENT_FUNCTION();
```

Comment:
`ImuSensor` is not an Unreal object, and therefore this constructor will never get called automatically by Unreal. Our convention is to only use `SP_LOG_CURRENT_FUNCTION()` on Unreal entry points, and only in locations where it won't be called every frame.

Reply:
That makes sense, updated this